project/SparkBuild.scala

```diff
@@ -929,7 +929,12 @@ object SparkProtobuf {
     (Test / PB.protoSources) += (Test / sourceDirectory).value / "resources" / "protobuf",
 
     (Test / PB.targets) := Seq(
-      PB.gens.java -> target.value / "generated-test-sources"
+      PB.gens.java -> target.value / "generated-test-sources",
+      PB.gens.descriptorSet -> target.value / "generated-test-sources/descriptor-set-sbt.desc",
```

Comment:
@LuciferYang this PR is an attempt at generating descriptor files at build time. it is not clear if there is a way to make SBT create one descriptor file for each proto file.

Reply:
I'll take a look at this one tomorrow

Reply:
I think it is ok， Whether they are in multiple files or all in one